pkgs/by-name/am/amber-lang/package.nix

```diff
@@ -0,0 +1,76 @@
+{ lib,
+  fetchFromGitHub,
+  rustPlatform,
+  bc,
+  bash,
+  makeWrapper,
+  runCommand,
+  amber-lang
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "amber-lang";
+  version = "0.3.1-alpha";
+
+  src = fetchFromGitHub {
+    owner = "Ph0enixKM";
+    repo = "Amber";
+    rev = version;
+    hash = "sha256-VSlLPgoi+KPnUQJEb6m0VZQVs1zkxEnfqs3fAp8m1o4=";
+  };
+
+  cargoHash = "sha256-NzcyX/1yeFcI80pNxx/OTkaI82qyQFJW8U0vPbqSU7g=";
+
+  nativeBuildInputs = [ bc bash ];
+  buildInputs = [ makeWrapper ];
+
+  preConfigure = ''
+    substituteInPlace src/compiler.rs \
+      --replace "/bin/bash" "${bash}/bin/bash"
+  '';
+
+  # FIXME: currently most tests are broken, this program is still under active
+  # development, so this is /okay/, but we should occasionally check if the
+  # situation improves.
+  checkFlags =
```

Comment:
checkFlags can be a list instead?

Reply:
with the suggested change from https://github.com/NixOS/nixpkgs/pull/313774#discussion_r1611077318, I can remove all the `checkFlags` and have the tests pass successfully.
![Uploading image.png…]()

Reply:
I just tested that, and also works for me, very nice :D